src/Analyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer.Test/NameAnalyzerTests.cs

```diff
@@ -0,0 +1,348 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using TestHelper;
+using DurableFunctionsAnalyzer;
+
+namespace DurableFunctionsAnalyzer.Test
+{
+    [TestClass]
+    public class NameAnalyzerTests : CodeFixVerifier
+    {
+
+        [TestMethod]
+        public void Should_not_trigger_on_empty()
+        {
+            var test = @"";
+
+            VerifyCSharpDiagnostic(test);
+        }
+        [TestMethod]
+        public void Should_not_find_any_issue_with_correctly_named_functions()
+        {
+            var test = @"using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace ExternalInteraction
+{
+    public static class HireEmployee
+    {
+        [FunctionName(""HireEmployee"")]
+        public static async Task<Application> RunOrchestrator(
+            [OrchestrationTrigger] DurableOrchestrationContext context,
+            ILogger log)
+            {
+                var applications = context.GetInput<List<Application>>();
+                var approvals = await context.CallActivityAsync<string>(""ApplicationsFiltered"", ""An approval"");
+                log.LogInformation($""Approval received. {approvals.Count} applicants approved"");
+                return approvals.OrderByDescending(x => x.Score).First();
+            }
+
+        [FunctionName(""ApplicationsFiltered"")]
+        public static async Task<string> Run(
+            [ActivityTrigger(""approval-queue"")] String approval,
+            [OrchestrationClient] DurableOrchestrationClient client)
+        {
+            await client.RaiseEventAsync(approval.InstanceId, ""ApplicationsFiltered"", approval.Applications);
+        }
+    }
+}";
+            
+
+            VerifyCSharpDiagnostic(test);
+
+            //        var fixtest = @"
+            //using System;
+            //using System.Collections.Generic;
+            //using System.Linq;
+            //using System.Text;
+            //using System.Threading.Tasks;
+            //using System.Diagnostics;
+
+            //namespace ConsoleApplication1
+            //{
+            //    class TYPENAME
+            //    {   
+            //    }
+            //}";
+            //        VerifyCSharpFix(test, fixtest);
```

Comment:
Can this commented-out code be removed? I believe there were a few other places with commented-out code like this.